roles/uperf/tasks/start_client.yml

```diff
@@ -0,0 +1,73 @@
+---
+
+- name: Get pod info
+  k8s_facts:
+    kind: Pod
+    api_version: v1
+    namespace: '{{ operator_namespace }}'
+    label_selectors:
+      - type = uperf-bench-server-{{ trunc_uuid }}
+  register: server_pods
+
+- name: Generate uperf xml files
+  k8s:
+    definition: "{{ lookup('template', 'configmap.yml.j2') | from_yaml }}"
+
+- block:
+  ### <POD> kind 
+  - name: Start Client(s) w/o serviceIP
+    k8s:
+      definition: "{{ lookup('template', 'workload.yml.j2') | from_yaml }}"
+    vars: 
+      resource_item: "{{ server_pods.resources }}"
+    when: workload_args.serviceip is defined and not workload_args.serviceip|default('false') and server_pods.resources|length > 0
```

Comment:
We don't need a default value here as you are checking if it's defined before checking for false, correct?

Reply:
I think it could be shortened with: `not workload_args.serviceip|default('false', True) and server_pods.resources|length > 0`

Reply:
Yes, it was extraneous. Fix done.